docs/guides/self-signed-ca-certs.md

```diff
@@ -0,0 +1,50 @@
+---  
+title: Fetch Packages from Registry with Self-Signed CA Certificate  
+toc: true  
+weight: 270  
+indent: true  
+---  
+
+# Overview of Crossplane for Registry with Self-Signed CA Certificate  
+
+Crossplane uses the go-containerregistry remote pkg as the client to fetch
+package manifests and images from OCI registries. It uses the 
+http.DefaultTransport, meaning that it loads the default system certificates.
+This precludes fetching packages from private registries that may use certs
+from a self-signed CA.
+
+The default transport can be overridden with one that passes additional certs
+to the TLSConfig. This allows for self-signed certificate authorities to be
+trusted.
+
+Crossplane needs to be installed via the Helm chart with the 
+`registryCaBundleConfig.name` and `registryCaBundleConfig.key` parameters 
+defined. See [Install Crossplane](https://crossplane.io/docs/v1.6/reference/install.html#uninstalling-the-chart).
+
+## Conifgure
+
+1. Create a CA Bundle (A file containing your Root and Intermediate 
+certificates in a specific order). This can be done with any text editor or 
+from the command line, so long as the resulting file contains all required crt 
+files in the proper order. In many cases, this will be either a single 
+self-signed Root CA crt file, or an Intermediate crt and Root crt file.The 
+order of the crt files should be from lowest to highest in signing order. 
+For example, if you have a chain of two certificaes below your Root 
+certificate, you place the bottom level Intermeidate cert at the beginning of 
+the file, then the Intermediate cert that singed that cert, then the Root cert 
+that signed that cert.
+
+2. Save the files as `[yourdomain].ca-bundle`.
+
+3. Create a Kubernetes ConfigMap in your Crossplane system namespace:
+
+```
+kubectl -n [Crossplane system namespace] create cm ca-bundle-config /
+--from-file=ca-bundle=./[yourdomain].ca-bundle
+```
+
+4. Set the `registryCaBundleConfig.name` Helm chart parameter to 
+`ca-bundle-config` and the `registryCaBundleConfig.key` parameter to 
+`ca-bundle`.
```

Comment:
Would you mind adding the snippet below that would need to be included in `values.yaml` for this to work?

Reply:
Sorry, I'm not clear on which snippet.

Reply:
@natereid72 `registryCaBundleConfig.name` and `registryCaBundleConfig.key` are configuration values passed to the helm chart. They can be passed via command (e.g. `--set registryCaBundleConfig.key=<value>`) or via a `values.yaml` provided with `-f`. I was just thinking it would be useful to folks to have a snippet provided that they could copy, for example:

```
registryCaBundleConfig:
  name: ca-bundle-config
  key: ca-bundle
```

Reply:
ah ok, thanks for the clarification. I didn't specify ways to apply parameters via helm because that's a helm detail. I figure people can rtm on helm if it's confusing, but most that have used helm will know. There are so many possible ways to deploy a helm chart (e.g. all the CD tools) that it seems cleaner to just say do that, and then someone can lookup the docs on helm if needed. Happy to add, just wanted to float this to be sure.

Reply:
Given this is a guide that is helping folks run Crossplane with this specific configuration, and that these configuration values are specific to Crossplane (even if they are provided via Helm), I think it would be useful to add 👍🏻